runtime/tools/python/test/test_run.py

```diff
@@ -47,6 +47,7 @@ def test_flatbuffer_cmd():
     ), f"one of more tests failed in={test_flatbuffer_cmd.__name__}"
 
 
+@pytest.mark.skip
 def test_dir_flatbuffer():
```

Comment:
Why are we disabling test_dir_flatbuffer? Shouldn't we make sure that this test always passes as it runs all the Silicon tests on device? I don't think this test uses any perf features?

Reply:
@tapspatel, why did we have to skip this one?

Reply:
I've tested this today, seems to be working (locally and on [git](https://github.com/tenstorrent/tt-mlir/actions/runs/11542583929/job/32126686476)). I added those two tests back, along with one more revert for metal build flavor which breaks ttrt locally.

@jnie-TT can you please remove block so we can check in tmrw.